docs/requirements.txt

```diff
@@ -3,3 +3,4 @@
 # under `install_requires` in `setup.cfg` is also listed here!
 sphinx>=3.2.1
 sphinx_rtd_theme
+pyscaffold
```

Comment:
To avoid pre-commit (and the coverage task on the CI) failing, I suppose pyscaffold should go to the top of the list...

Yeah, sometimes pre-commit is a bit annoying, but I love having it around...

Reply:
Thanks :-) It's funny though that the pre-commit plugin solves it by putting it even before the comments.